q2cli/builtin/tools.py

```diff
@@ -715,21 +715,34 @@ def cache_create(cache):
               type=click.Path(exists=True, file_okay=False, dir_okay=True,
                               readable=True),
               help='Path to an existing cache to remove the key from.')
-@click.option('--key', required=True,
-              help='The key to remove from the cache.')
+@click.option('--key', required=True, multiple=True,
+              help='The key to remove from the cache. Pass multiple times to '
+                   'remove multiple keys.')
```

Comment:
```suggestion
              help='The key to remove from the cache. Pass `key` parameter multiple times to '
                   'remove multiple keys.')
```
I feel like this could be alittle clearer